packages/grafana-ui/src/components/PieChart/PieChart.tsx

```diff
@@ -47,9 +54,26 @@ export const PieChart: FC<PieChartProps> = ({
   onSeriesColorChange,
   width,
   height,
+  eventBus,
   ...restProps
 }) => {
   const theme = useTheme();
+  const [state, setState] = useSetState<PieChartState>();
```

Comment:
no need for an object state, you can just have `const [highlightedTitle, setHighlightedTitle] = useSetState<string>();`